test/inductor/test_cpu_select_algorithm.py

```diff
@@ -560,6 +560,7 @@ def forward(self, x, other, other2):
 
     @inductor_config.patch({"freezing": True})
     @patches
+    @torch._dynamo.config.patch(inline_inbuilt_nn_modules=True)
```

Comment:
Perhaps change the `inline_inbuilt_nn_modules` in the patch to make it effects for all the UT in this test file.